run_single_op.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+# Test version - runs only relu operation
```

Comment:
Do we want to commit this as I think this is just a personal testing script right? correct me if I'm wrong. Otherwise, just rename it to smoke_test.sh / run_refu.sh